.rubocop.yml

```diff
@@ -21,18 +21,19 @@ SpaceInsideBrackets:
 Style/Alias:
   Enabled: false
 
-Style/IndentArray:
-  Enabled: false
-
-Style/IndentHash:
+Layout/IndentArray:
   Enabled: false
 
 HashSyntax:
   EnforcedStyle: ruby19
 
-LineLength:
+Metrics/LineLength:
   Max: 120
 
+Metrics/ModuleLength:
+  CountComments: false  # count full line comments?
+  Max: 130
```

Comment:
There is one file that fails validation for Metrics/ModuleLength, validations/translations.  _I'm_ OK with this value, but happy to discuss.